src/crystal/readline/lib_readline.cr

```diff
@@ -0,0 +1,21 @@
+module Crystal
+  @[Link("readline")]
```

Comment:
Does this mean `readline` would be moved back in to stdlib? (https://github.com/crystal-lang/crystal-readline)

Reply:
It depends on whether we'll want to definitely use readline for this (I guess yes, because not using it is a bit annoying when typing.) But it would live inside the `Crystal` namespace, hidden from the std, just like the current `Markdown` lib.